packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -89,12 +89,12 @@
                 "properties": {
                   "text": {
                     "type": ["null", "string"],
-                    "description": "Header text.",
+                    "description": "Header text. This can contain {{...}} expressions.",
```

Comment:
@mblumreich I forgot to ping you initially -- all these `description`s will show as hover text in the editor. Let me know if they need updates.